test-runtime/index.html

```diff
@@ -8,7 +8,14 @@
     <div id="vis"></div>
 
     <script type="text/javascript">
+      // global
+      var view = null;
```

Comment:
I just added this to make it explicit that the view is global and so that the `if (view)` below does not crash when the variable is undefined.